pkgs/development/libraries/libvmaf/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, autoconf, automake, intltool, libtool, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "libvmaf";
+  version = "1.3.15";
+
+  src = fetchFromGitHub {
+    owner = "netflix";
+    repo = "vmaf";
+    rev = "v${version}";
+    sha256="13ajbcidngjvgl0rr7l0mb43h651p5pqj3d1linrfk9c222b9fs3";
+  };
+
```

Comment:
not a big deal, but i would also add multiple outputs
```
  outputs = [ "out" "dev" ];
```
cleans up the outputs a bit, but not by much